reijnaers2021re/openreview/packages.tex

```diff
@@ -0,0 +1,30 @@
+%\usepackage[utf8]{inputenc}
+
+\usepackage{biblatex}
+\addbibresource{bibliography.bib}
+
+\usepackage[T1]{fontenc} 
```

Comment:
Please remove the import to `fontenc` as it breaks the ReScience fonts

Reply:
I don't understand how I could have missed this. Fixed!